src/applications/yellow-ribbon/actions/index.js

```diff
@@ -35,6 +36,11 @@ export const toggleShowMobileFormAction = () => ({
   type: TOGGLE_SHOW_MOBILE_FORM,
 });
 
+// ============
+// Toggle alert tool tip
+// ============
+export const toggleToolTip = () => ({ type: TOGGLE_TOOL_TIP });
```

Comment:
Since the `SearchForm` has its own tool tip controlled by local state, it seems like this name could be clearer, like `toggleSearchResultsToolTip` or something like that